packages/yourloops/components/patient/add-patient-dialog.tsx

```diff
@@ -62,16 +60,15 @@ export const AddPatientDialog: FunctionComponent<AddDialogProps> = ({ onClose, o
   const alert = useAlert()
   const patientHook = usePatientContext()
   const { t } = useTranslation()
-  const { teams, getTeam } = useTeam()
   const [email, setEmail] = useState<string>('')
-  const [teamId, setTeamId] = useState<string>('')
   const [errorMessage, setErrorMessage] = useState<string | null>(null)
   const [inProgress, setInProgress] = useState<boolean>(false)
+  const { selectedTeam } = useSelectedTeamContext()
 
   const isValidEmail = (mail = email): boolean => mail.length > 0 && REGEX_EMAIL.test(mail)
 
   const handleClickAdd = async (): Promise<void> => {
-    if (isValidEmail() && teamId.length > 0) {
+    if (isValidEmail() && selectedTeam.id.length > 0) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏I think these checks are useless here because already binded to the disabled state of the button. Also I think this dialog could be simplified, `handleVerifyEmail` method can be deleted, it's useless since the email is already checked each type event with the `onChange` prop.